modules/govuk/manifests/apps/asset_manager.pp

```diff
@@ -88,7 +88,40 @@
       location ~ /raw/(.*) {
         internal;
         alias /var/apps/asset-manager/uploads/assets/$1;
-      }',
+      }
+
+      location = /cloud-storage-proxy/ {
+        # Prevent requests to this location from outside nginx
+        internal;
+
+        # Extract download host and URL from upstream HTTP headers
+        set $cloud_storage_host $upstream_http_x_cloud_storage_host;
+        set $download_url $upstream_http_x_cloud_storage_url;
+
+        # The X-CLOUD-STORAGE-URL header contains a signed URL for the asset on
```

Comment:
I agree that this comment is closely tied with the fact that we're sending presigned URLs and that the implementation still works if we send public URLs. I'm just not sure how best to convey that!